src/main/java/DoneCommand.java

```diff
@@ -0,0 +1,16 @@
+public class DoneCommand extends Command{
+    private final int n;
+
+    DoneCommand(Tasks[] tasks, int n) {
+        super(tasks);
+        this.n = n;
+    }
+
+    public void markDone() {
+        displayLine();
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println("[" +tasks[n-1].displayTaskType()+ "] [\u2718] " + tasks[n-1].getDescription());
+        displayLine();
+        tasks[n-1].setDone();
```

Comment:
This is just a suggestion. The script executes the "setDone" method after displaying the message "I have marked this task as done". The sequence is ok when everything works fine as there are only probably several milliseconds of delay. But when the "setDone" method causes an error, the program would crash after the message "I have marked this task as done". This would be a bit misleading.